Makefile

```diff
@@ -1,5 +1,13 @@
+# #############################################################################
+# Simple Makefile to build and test Lightweight Logging Library (l3) sources
+# #############################################################################
 test: test.c l3.c l3.S l3.h
-	gcc -Wall -o test -g -O3 test.c l3.c l3.S l3.h
+	pylint l3_dump.py
+	gcc -Wall -o test -g -O3 test.c l3.c l3.S
 	./test
-	python l3_dump.py /tmp/l3_test test
-	python l3_dump.py /tmp/l3_small_test test
+	@echo
```

Comment:
Add blank lines between every new command run, so the chunks of output are more visible in stdout.

Also on L9 below, we run the Py script w/o args, just to be sure that the help/usage message is still coming out correctly.